Automattic/vip-cli
# Fallback to the first element in the array of environments when fetching the app info for dev env wizard and no environment type is passed

## Description

Currently, if a `@ID `notation is used (without the environment type is used) we don't fetch the needed information.  We should match the behavior where omitting the env fetched the production info


## Steps to Test

Outline the steps to test and verify the PR here.

Example:

1. Check out PR.
1. Run `npm run build`
1. Try `vip @1337 dev-env create`
1. Verify the correct properties are picked for the environment
